src/components/Search/SearchInput.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+
+import Grid from '@mui/material/Grid';
+import TextField from '@mui/material/TextField';
+import { connect } from 'react-redux';
+
+import store from '../../common/store';
+import { fetchRecentRevisions } from '../../thunks/searchThunk';
+import { Revision, State } from '../../types/state';
+import { handleChangeSearch } from '../../utils/searchViewHelper';
+import SearchResultsList from './SearchResultsList';
+
+class SearchInput extends React.Component<SearchInputProps> {
+  componentDidMount() {
+    const repo = store.getState().search.repository;
+    store.dispatch(fetchRecentRevisions(repo));
+  }
```

Comment:
fetch recent revisions on mount now that we have a default repo set. this will display recent revisions when focusing input.

Reply:
It would be better to have behavior changes in a separate PR, to make it easier to review.

I believe this PR should be technical only and not bring behavior changes.